Blizzard.IO.RabbitMQ/Rpc/INetqRpcRabbitConnection.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Blizzard.IO.RabbitMQ.Entities.Rpc;
+using EasyNetQ;
+using System;
+
+namespace Blizzard.IO.RabbitMQ.Rpc
+{
+    public interface INetqRpcRabbitConnection<TReturnType>
```

Comment:
Why don't we implement these methods as extension methods on `IBus` exactly like `R.I.P RabbitConnection`'s methods ?

Reply:
Cause the class implementing this interface should keep state

Reply:
What state? it seems that this class only configures and creates the bus

Reply:
And saves the `RpcMessageType` the bus is configured to

Reply:
The user may change it on runtime?

Reply:
Nope